src/vivarium_gates_child_iv_iron/data/loader.py

```diff
@@ -817,3 +818,21 @@ def load_maternal_bmi_anemia_excess_shift(key: str, location: str) -> pd.DataFra
         .sort_index()
     )
     return excess_shift
+
+
+def load_neonatal_lri_csmr(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.AFFECTED_UNMODELED_CAUSES.NEONATAL_LRI_CSMR:
+        raise ValueError(f"Unrecognized key {key}")
+
+    data = load_standard_data(data_keys.LRI.CSMR, location)
+    data.loc[data.index.get_level_values('age_start') >= 0.076712, :] = 0
+    return data
```

Comment:
Would it be useful to add this `0.076712` to a constants file?

Reply:
Sure, can do.